src/Javabasic.java

```diff
@@ -1,6 +1,100 @@
+            System.out.print( "4. Removing special Characters : " );  //printing output
+            System.out.println(sample.replaceAll(("[^a-zA-Z0-9]"), " "));  }  //print result string without Special Character
+
+        public static void extractnumber(){  // method for extracting number from string
+            String sample= "The cost of iPhone is £1,250";  // sample string
+            System.out.print( "5. The cost of iPhone is  : " );  //printing output
+            System.out.println(sample.replaceAll("[^0-9]", ""));     }  //printing result for extracting number from string
+
+        public static void inttonum(){  // method for integer to double
+            int sample = 450;  // sample string
+            System.out.print( "6. value of integer to number is : " );  //printing output
+            System.out.println((double)(sample));    }  //printing result for integer to double
+
+        public static void stringtoint(){  // method created to convert String value to Integer
+            String sample = "125";  // sample string
+            System.out.print( "7. String value to integer will be : " );  //printing output
+            System.out.println(Integer.parseInt(sample)); } //Result for String to integer
+
+        public static void doubletostring (){  // method created to convert double value to String
+            double sample = 123.45;  //sample string
+            System.out.print( "8. Value from Double to String will be : ");   //printing output
+            System.out.println(String.valueOf(sample));     }  // print Result for double to string
+
+        public static void stringtodouble (){  // this method will convert string value to double
+            String sample = "560.25";   // sample String
+            Double value = Double.parseDouble(sample);
+            System.out.print( "9. Value from String to Double is : " );  //printing output
+            System.out.println(value);     }  // print result for string to double
+
+        public static void divisionindecimal(double x, double y){  //method to get answer in decimal value
+            String value1 =  "1";
+            String value2 = "3";
+            System.out.print( "10. Division of 1/3 in decimal will be : " );  //printing output
+            System.out.println(x/y);
+            System.out.println(Double.parseDouble(value1) + Double.parseDouble(value2));}  // print result in decimal value
+
+        public static void replacecurrencysymbol() {  //method to replace currency symbol from $ to £
+            String sample = "$250.00";  //sample string
+            System.out.print("11. When currency symbol replaced from $ to £ is : "); // printing output
+            System.out.println(sample.replace("$", "£")); }// print result with replacement currency symbol
+
+        public static void extractfrommid(String x) { // this method will extract middle character from string
```

Comment:
Good method name are starts with small letter but every new word should begin with capital letter e.g. extractMiddleOfString